src/getHumanAge.test.js

```diff
@@ -7,4 +7,28 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [0, 0] for ages 0, 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] for ages 14, 14', () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] for ages 15, 15', () => {
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+  });
```

Comment:
The test description 'should return [0, 0] for ages 15, 15' is incorrect. It should be 'should return [1, 1] for ages 15, 15' to match the expected output `[1, 1]`.